### ISSUE TITLE
Wrong string representation of the List<T>

### ISSUE THREAD

**Issue by dmitriifilonenko** [ThreeMammals/Ocelot#678]
In method Ocelot.Authorisation.Authorise(...):
if claims do not contain required value then an ErrorResponse will be created:

```
return new ErrorResponse<bool>(new ClaimValueNotAuthorisedError(
                                $"claim value: {values.Data} is not the same as required value: {required.Value} for type: {required.Key}"));
```
But values.Data is of type List<string>. So implicit convertation of values.Data to string gives _"System.Collections.Generic.List`1[System.String]"_ instead of sequence of items in the list like "Item1, Item2, Item3".

As a result I see the following error in the log whenever claims in request do not contain a claim with the name "role":

> claim value: System.Collections.Generic.List`1[System.String] is not the same as required value: Admin for type: role

## Specifications

  - Version: 12.0.1

**Comment by briansantura** [comment]
@dmitriifilonenko I have created a pull request for this issue.

**Comment by raman-m** [comment]
Fixed by #711
